src/lib/bottom-sheet/bottom-sheet.ts

```diff
@@ -143,6 +145,10 @@ export class MatBottomSheet {
 
     injectionTokens.set(MatBottomSheetRef, bottomSheetRef);
     injectionTokens.set(MAT_BOTTOM_SHEET_DATA, config.data);
+    injectionTokens.set(Directionality, {
+      value: config.direction,
+      change: observableOf()
+    });
```

Comment:
We should only set this token if there's not already a `Directionality` in the `userInjector`

Reply:
Done.